src/pages/LoginPage.js

```diff
@@ -17,6 +17,7 @@ const LoginPage = () => {
       .then((res) => res.json())
       .then((data) => setUser(data));
   }, []);
+  localStorage.setItem("allUsers", JSON.stringify(users));
```

Comment:
Do not store all users in the local storage. It is not a good practice. Ideally FE should not store any data. everything should be on the BE and database, and retrieved via api calls

Reply:
resolved